tin/tests/create_users.py

```diff
@@ -0,0 +1,47 @@
+from ..apps.users.models import User
+
+
+def make_users(password: str, debug: bool = True) -> None:
+    # fmt: off
+    users = [
+        #[username, id, full_name, first_name, last_name, email, is_teacher, is_student, is_staff, is_superuser]
+        ["2020jbeutner", 33538, "John Beutner", "John", "Beutner", "2020jbeutner@tjhsst.edu", False, True, False, False],
+        ["2020fouzhins", 33797, "Theo Ouzhinski", "Theo", "Ouzhinski", "2020fouzhins@tjhsst.edu", False, True, False, False],
+        ["2024kshankar", 1000891, "Krishnan Shankar", "Krishnan", "Shankar", "2024kshankar@tjhsst.edu", False, True, False, False],
+        ["2027adeshpan", 1906054, "Aarush Deshpande", "Aarush", "Deshpande", "2027adeshpande@tjhsst.edu", False, True, False, False],
+        ["pcgabor", None, "Peter Gabor", "Peter", "Gabor", "pcgabor@fcps.edu", True, False, False, False],
```

Comment:
I think you should replace the above with generic users (`2024student`, `2025student`, `astaff`, `bstaff`, etc.), since it would probably be easier to work with, especially for dev environment stuff.